models/gif_cond_exp_multisynapse.h

```diff
@@ -0,0 +1,467 @@
+  firing intensity:
+
+  lambda(t) = lambda_0 * exp[(V(t)-V_T(t)/Delta_V)]
+
+  where V_T(t) is a time-dependent firing threshold:
+
+  V_T(t) = V_T_star + gamma_1(t) + gamma_2(t) + ... + gamma_m(t)
+
+  where gamma_i is a kernel of spike-frequency adaptation (sfa), and the neuron
+  model can have arbitrary number of them.
+  Dynamic of each gamma_i is described by:
+
+  tau_gamma_i*d{gamma_i}/dt = -gamma_i
+
+  and in case of spike emission, its value increased by a constant (which can be
+  positive or negative):
+
+  gamma_i = gamma_i + q_gamma_i  (in case of spike emission).
+
+  Note that in the current implementation of the model (as described in [1] and
+  [2]) the values of eta_i and gamma_i are affected immediately after spike
+  emission. However, GIF toolbox (http://wiki.epfl.ch/giftoolbox) which fits
+  the model using experimental data, requires a different set of eta_i and
+  gamma_i. It applies the jump of eta_i and gamma_i after the refractory period.
+  One can easily convert between q_eta/gamma of these two approaches:
+  q_eta_giftoolbox = q_eta_NEST * (1 - exp( -tau_ref / tau_eta ))
+  The same formula applies for q_gamma.
+
+  On the postsynapic side, there can be arbitrarily many synaptic time constants
+  (gif_psc_exp has exactly two: tau_syn_ex and tau_syn_in). This can be reached
+  by specifying separate receptor ports, each for a different time constant. The
+  port number has to match the respective "receptor_type" in the connectors.
+
+  The shape of synaptic conductance is exponential.
+
+  Parameters:
+  The following parameters can be set in the status dictionary.
+
+  Membrane Parameters:
+    C_m        double - Capacity of the membrane in pF
+    t_ref      double - Duration of refractory period in ms.
+    V_reset    double - Reset value after a spike in mV.
+    E_L        double - Leak reversal potential in mV.
+    g_L        double - Leak conductance in nS.
+    I_e        double - Constant external input current in pA.
+
+  Spike adaptation and firing intensity parameters:
+    q_stc      vector of double - Values added to spike-triggered currents (stc)
+                                  after each spike emission in nA.
+    tau_stc    vector of double - Time constants of stc variables in ms.
+    q_sfa      vector of double - Values added to spike-frequency adaptation
+                                  (sfa) after each spike emission in mV.
+    tau_sfa    vector of double - Time constants of sfa variables in ms.
+    Delta_V    double - Stochasticity level in mV.
+    lambda_0   double - Stochastic intensity at firing threshold V_T in 1/s.
+    V_T_star   double - Minimum threshold in mV
+
+  Synaptic parameters
+    taus_syn   vector of double - Time constants of the synaptic conductance
+                                  in ms.
+    E_ex       double - Excitatory reversal potential in mV.
+    E_in       double - Inhibitory reversal potential in mV.
```

Comment:
@hesam-setareh @Silmathoron Sorry for not updating on this earlier. In the [Developer VC 19 Sep](https://github.com/nest/nest-simulator/wiki/2016-09-19-Open-NEST-Developer-Video-Conference) we decided that for conductance based neurons, each rport should have one input channel and one reversal potential. So instead of `E_ex` and `E_in`, where should be a vector of reversal potentials the same size as `taus_syn`. 

**BUT**: I just recently merged #439 and #474 with multisynapse models that do not implement the new logic either. So I would not consider this a blocker for this PR either. I will create new issues and we should then try to fix this through new PRs asap.